goodtablesio/blueprints/github.py

```diff
@@ -1,59 +1,55 @@
+import re
 import os
 import subprocess
 import tempfile
 import uuid
 import logging
-
 from flask import Blueprint, request, abort
-
-from goodtablesio import handlers
-
-
+from .. import helpers
 log = logging.getLogger(__name__)
 
 
-TABULAR_EXTENSIONS = ['csv', 'xls', 'xlsx', 'ods']
+SECRET = 'HUGI#6A2X|e{.Rkn`zg?a!`/9(&(Y7WYQqW#5.(&][s&-W(A8y;85pC:&;H<v*aw'
```

Comment:
shouldn't `SECRET` be in the config object, and also thereby settable from env var?